source/material_model/equation_of_state/thermodynamic_table_lookup.cc

```diff
@@ -144,7 +144,12 @@ namespace aspect
           }
       }
 
-
+      template <int dim>
+      const std::vector<std::string> &
+      ThermodynamicTableLookup<dim>::get_phase_names_one_lookup(unsigned int lookup_index) const
```

Comment:
Here and elsewhere, mark input arguments to function by `const`:
```suggestion
      ThermodynamicTableLookup<dim>::get_phase_names_one_lookup(const unsigned int lookup_index) const
```

Reply:
Think I've caught these with latest updates. Resolving after I implement latest changes in visco_plastic.cc if something slipped past me.